common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -603,11 +601,10 @@ def oauth_params(self, custom_parameters, client_key, client_secret):
             except AttributeError:
                 self.user_username = ""
 
-        if self.open_in_a_new_page:
```

Comment:
Can you clarify because I'm confused here. Why was this originally here? What was the intention? Why are you removing it, instead of just setting this to "true"?